### PR TITLE
updated lead

### PR DESCRIPTION
**Pull request by DineshKoushik**

### CODE REVIEW

**Comment by DineshKoushik** [comment]
@vivianha534 , Hey can you check if this works?

File: _data/classes.yml
Comment (vivianha534) [inline _data/classes.yml:66]: Please change the hash from aubramonian to aasmani as well. The hash is based off of the person first name and last name (ie Ava's last name is asmani so her hash is aasmani). By changing the hash, this will change where the link directs to c:

**Review by vivianha534 — APPROVED** [review]
You're almost there! After you change the hash feel free to squash and merge your pull request!

**Comment by DineshKoushik** [comment]
@vivianha534 , Hi again can you check the new PR?

**Comment by vivianha534** [comment]
@DineshKoushik looks great! Thanks for your work and feel free to squash merge when you're ready!

**Comment by DineshKoushik** [comment]
@vivianha534 , I don't know what is a squash merge, would you mind explaining it?

**Comment by vivianha534** [comment]
@DineshKoushik that's a great question! Squash merge's are a type of merge in git. Basically squash merging a branch squashes all of the commits to a branch into one single commit before the branch is merged into master. Sqaush merge allows you to create a more condensed git history. In order to squash merge your pull request, you should see a green squash and merge button near the bottom of the page! If this doesn't make sense feel free to let me know and I can try to explain it in a different way.

Here's some more info about squash merging: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#:~:text=When%20you%20select%20the%20Squash,merged%20into%20the%20default%20branch.

**Comment by DineshKoushik** [comment]
@vivianha534 , thank you for the info, does this work?

**Comment by vivianha534** [comment]
@DineshKoushik I didn't see any changes on my end so I'm not entirely sure what you're asking about when you asks if this works? But just to further clarify, you can find the squash and merge circled in the image below
Squash

**Comment by DineshKoushik** [comment]
@vivianha534 , unfortunately I cannot see that squash and merge button, but I squashed through git yesterday.
image

**Comment by vivianha534** [comment]
@DineshKoushik hmm, it doesn't look like your squash merge worked, but I can go ahead and squash it for you

**Pull request merged**
